### PR TITLE
Added pydocs to mrc.localization.color.kmeans

### PR DESCRIPTION
**Pull request by Maheliusz**

### CODE REVIEW

File: mrc/localization/color/kmeans/predicates.py [hunk 1]
```diff
@@ -2,7 +2,26 @@
 
 
 class GaussianHSVPredicate:
+    """
+    Class representing predicate based on gaussian distribution.
+
+    Intended for internal usage.
+    """
+
     def __call__(self, hue, range_):
```
Comment (Lukasz1928) [inline mrc/localization/color/kmeans/predicates.py]: Wouldn't it be better to pass range as tuple(a, b) or sth similar?

File: mrc/localization/color/kmeans/analyser.py [hunk 2]
```diff
@@ -6,18 +6,55 @@
 
 
 class Predicates:
```
Comment (Lukasz1928) [inline mrc/localization/color/kmeans/analyser.py]: It's not a very beautiful solution. We could consider changing it.

File: mrc/localization/color/kmeans/analyser.py [hunk 3]
```diff
@@ -6,18 +6,55 @@
...
         self._color_values = color_values
 
     def analyse_chunk(self, image_chunk: ndarray) -> dict:
+        """
+        Function counting probabilities of all color occurences in given input.
+
+        Parameters
+        ----------
+        image_chunk : :obj:`ndarray`
+            Fragment of RGB image read by using OpenCV
```
[the remark this answers is not in the document]
Comment (Maheliusz) [reply mrc/localization/color/kmeans/analyser.py]: Fixed

File: mrc/localization/color/kmeans/predicates.py (hunk 1 above)
Comment (Maheliusz) [reply mrc/localization/color/kmeans/predicates.py]: Changed

File: mrc/localization/color/kmeans/analyser.py (hunk 2 above)
Comment (Maheliusz) [reply mrc/localization/color/kmeans/analyser.py]: We could, but until we figure out something nicer let's leave it at that

File: tests/localization/color/kmeans/common.py [hunk 4]
```diff
@@ -8,7 +10,7 @@
     ((541.1138305664062, 43.25203323364258), 3.749695301055908),
     ((392.18328857421875, 29.525123596191406), 8.324203491210938)]
 
-image = cv2.imread('tests/localization/color/kmeans/test_image.jpg')
+image = cv2.imread(os.path.abspath("tests/resources/localization/color/kmeans/test_image.jpg"))
```
Comment (Lukasz1928) [inline tests/localization/color/kmeans/common.py]: Previous version seems to be OK.

File: tests/localization/color/kmeans/common.py [hunk 5]
```diff
@@ -8,7 +10,7 @@
...
 
 colors_analyzed = [
     {'red': 3.883003111115027e-269, 'yellow': 1.2664165549094177e-14, 'green': 0.48304802297166255},
```
Comment (Lukasz1928) [inline tests/localization/color/kmeans/common.py]: I'm not sure if exactly those values are expected ;)

Comment (Maheliusz) [reply tests/localization/color/kmeans/common.py]: It just works

File: tests/localization/color/kmeans/common.py (hunk 4 above)
Comment (Maheliusz) [reply tests/localization/color/kmeans/common.py]: Changed

**Pull request merged**